api/database/db.py

```diff
@@ -4,11 +4,6 @@
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import sessionmaker
 
-if os.environ.get("CI"):
```

Comment:
I would add the environment variable CI to the .env.example file so that we don't forget about it and if it needs to be a particular value.

Reply:
Unfortunately I'm not sure what you mean @sylviamclaughlin. 

The CI environment variable was only being checked to then set `connection_string` which doesn't appear to be used. I don't see any other place in the code that we do any kind of check to see if the code is in a CI environment - thus I'm not sure why at this point we need to have it in the `env.example` 

When we setup a proper CI environment - presumably we only need to run the tests - and when those are executed via `python -m pytest` in the api/ folder they do all of the test discovery and setup the test db correctly.

Reply:
I'm going to merge this request and then I'l chat with you re the CI env var in the coming days